source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -26,7 +26,7 @@ To investigate the emails we first need to [log in to the inbox][login].
 
 The current email addresses used for the fact checking process are:
 
-- factcheck@alphagov.co.uk
+- govuk-fact-check@digital.cabinet-office.gov.uk
 - govuk-fact-check-staging@digital.cabinet-office.gov
 - govuk-fact-check-integration@digital.cabinet-office.gov
```

Comment:
```suggestion
- govuk-fact-check-staging@digital.cabinet-office.gov.uk
- govuk-fact-check-integration@digital.cabinet-office.gov.uk
```

Just noticed that we should probably update these to be actual email addresses.

Reply:
🤣 well spotted!